### DISCUSSION TITLE
Another 404 error on website

### DISCUSSION THREAD

**Discussion by eufex** [apohl79/audiogridder#349] [General]
I think you need to check all links on the website as this is the 2nd one:

Link to discussions throws a 404.

https://audiogridder.com

**Comment by apohl79** [comment]
Thx dude! Website maintenance seems not so my thing! :-) much appreciated!

**Reply by apohl79** [reply]
strange, as the link in the menu as well as the one on the bugs site is working for me.

**Comment by CaBuffalo** [comment]
It works for me as well.

On Tue, Feb 2, 2021 at 11:03 AM Andreas Pohl <notifications@github.com>
wrote:

> strange, as the link in the menu as well as the one on the bugs site is
> working for me.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/349#discussioncomment-331957>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6G7K2CO4YO4JHP5C2DS5BD7ZANCNFSM4W7G7ZYA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by eufex** [comment]
To be clear, it's the first link under 'bug reports'
https://github.com/apohl79/audiogridder/discussions/new

**Comment by apohl79** [comment]
ok the problem is that the link works only if you are logged into github... thx. will fix it.
